impress-org/givewp
# Fix: remove and replace deprecated WP_Background_Process & WP_Async_Request

Resolves https://github.com/impress-org/givewp/issues/7064

## Description

This PR removes the deprecated version files of `WP_Background_Process` & `WP_Async_Request`. 
Also replacing `WP_Async_Request` with `WPAsyncRequest` in our `give-async-process`.

In [Feature: Add WordPressLibraries to manage WP_Background_Process #6856](https://github.com/impress-org/givewp/pull/6856) we had forked these files into namespaced versions and imported them into Give_Zapier to avoid fatal error conflicts occurring with 3rd party plugins and our add-on. At the time the decision was to keep the original files within core intact. Because the files are still being loaded we were aware of the possibility that conflicting errors may continue to occur. Recently a developer has reached out to help address the conflict errors within GhostKit, VisualPortfolio and Give due to the mismatched versions of the files being loaded.

## Visuals
<img width="2046" alt="Screen Shot 2023-11-15 at 5 41 39 AM" src="https://github.com/impress-org/givewp/assets/75056371/1fe4e27c-c624-4a3d-ac98-56f632d6516f">


## Testing Instructions
- Install GhostKit &/or VisualPortfolio
- Verify that no fatal errors occur.
## Pre-review Checklist
